frontend/src/metabase-types/api/notifications.ts

```diff
@@ -15,11 +12,10 @@ export type Channel = {
   details: Record<string, string>;
   enabled?: boolean;
   recipients?: NotificationRecipient[];
-  schedule_day?: null | ScheduleDayType;
-  schedule_frame?: null | "first";
-  schedule_hour?: number | null;
-  schedule_type?: ScheduleType;
-};
+} & Pick<
```

Comment:
I was getting some type errors [here](https://github.com/metabase/metabase/pull/46647/files#r1709807479) - since the `Channel` and `ScheduleSettings` types were separate, there was an error even though we treat those objects as similar things so I combined the type.